scripts/lint

```diff
@@ -8,7 +8,7 @@ export SOURCE_FILES="httpcore tests"
 
 set -x
 
-${PREFIX}autoflake --in-place --recursive $SOURCE_FILES
+${PREFIX}autoflake --in-place --recursive --remove-all-unused-imports --ignore-init-module-imports $SOURCE_FILES
```

Comment:
We can drop this option since all the imports in `__init__.py` files are used.

```suggestion
${PREFIX}autoflake --in-place --recursive --remove-all-unused-imports $SOURCE_FILES
```

Reply:
Sounds reasonable, thank you!